File: bot-script.py
import time
from telegram import Bot

# Replace 'YOUR_BOT_TOKEN' with the token you received from BotFather
TOKEN = '6234737963:AAF84w6FMwsSOrKO4aMW1yedqnoe_jAQ3aE'
CHANNEL_ID = 1001910479831  # Replace with your channel's username

def main():
    bot = Bot(token=TOKEN)
    while True:
        new_invite_link = bot.export_chat_invite_link(CHANNEL_ID)
        bot.send_message(chat_id=CHANNEL_ID, text=f"New link: {new_invite_link}")
        time.sleep(300)  # Wait for 5 minutes

if __name__ == '__main__':
    main()

File: bot_script.js
const TelegramBot = require('node-telegram-bot-api');
const cron = require('node-cron');

const botToken = '6234737963:AAF84w6FMwsSOrKO4aMW1yedqnoe_jAQ3aE';
const CHANNEL_ID = 1001910479831; // Replace with your channel's username

const bot = new TelegramBot(botToken, { polling: false });

cron.schedule('*/5 * * * *', async () => {
  try {
    const newInviteLink = await bot.exportChatInviteLink(CHANNEL_ID);
    console.log(`New link: ${newInviteLink}`);
  } catch (error) {
    console.error(`An error occurred: ${error}`);
  }
});
